apps/aurora-portal/src/client/routes/_auth/accounts/$accountId/projects/$projectId/compute/-components/Flavors/-components/useExtraSpecs.ts

```diff
@@ -0,0 +1,69 @@
+import { useState, useCallback } from "react"
```

Comment:
is `useExtraSpecs` used somewhere else except flavors specs component?

If not, you could move them in flavos specs component. There is no need of this hook. And I guess you also don't need the `useCallback`